python_modules/dagster/dagster/_core/storage/cloud_storage_compute_log_manager.py

```diff
@@ -62,10 +95,31 @@ def cloud_storage_has_logs(
         """Returns whether the cloud storage contains logs for a given log key."""
 
     @abstractmethod
+    def _upload_file_obj(
+        self, data: IO[bytes], log_key: Sequence[str], io_type: ComputeIOType, partial=False
+    ):
+        pass
+
     def upload_to_cloud_storage(
```

Comment:
i don't think this is technically part of the dagster public API but is there a way to do this in a way that wouldn't be a breaking change for other subclasses if somebody had written their own compute log manager?

an example would be to use some kind of mixin or subclass and use that in the subclasses that are doing this truncation